src/locales/cs.ts

```diff
@@ -0,0 +1,797 @@
+    dominic_de_guzman_priest: 'Sv. Dominika, kněze',
+    dyfrig_of_wales_bishop: 'Sv. Dubricia Velšského, biskupa',  // TODO: `Velšský` or `z Walesu`?
+    edmund_campion_priest: 'Sv. Edmunda Kampiána, kněze a mučedníka',
+    edmund_of_abingdon_bishop: 'Sv. Edmunde z Abingdonu, biskupa',
+    edward_the_confessor: 'Sv. Edvarda Vyznavače',
+    elijah_prophet: 'Sv. Eliáše, proroka',
+    elizabeth_ann_seton_religious: 'Sv. Alžběty Anny Setonové, řeholnice',  // TODO: Should we bohemise her name?
+    elizabeth_of_hungary_religious: 'Sv. Alžběty Uherské, řeholnice',
+    elizabeth_of_portugal: 'Sv. Alžběty Portugalské',
+    emeric_of_hungary: 'Sv. Emericha',
+    enda_of_aran_abbot: 'Sv. Endy, opata',
+    ephrem_the_syrian_deacon: 'Sv. Efréma Syrského, jáhna a učitele církve',
+    eric_ix_of_sweden_martyr: 'Sv. Erika IX. Švédského, Martyr',
+    etheldreda_of_ely_abbess: 'Sv. Eteldredy z Ely, opatky',
+    eugene_de_mazenod_bishop: 'Sv. Evžena de Mazenod, biskupa',
+    eoghan_of_ardstraw_bishop: 'Sv. Eoghana, biskupa',
+    eulalia_of_merida_virgin: 'Sv. Eulálie z Méridy, panny a mučednice',
+    eulogius_of_cordoba_bishop: 'Sv. Eulogia z Córdoby, biskupa',
+    eusebius_of_vercelli_bishop: 'Sv. Eusebia z Vercelli, biskupa',
+    eysteinn_of_nidaros_bishop: 'Sv. Eysteina z Nidarosu, biskupa',
+    ezequiel_moreno_bishop: 'Sv. Ezechiela Moreno, biskupa',
+    fabian_i_pope: 'Sv. Fabiána, papeže a mučedníka',
+    fachanan_of_kilfenora_bishop: 'Sv. Fachanana z Kilfenory, biskupa',
+    fachtna_of_rosscarbery_bishop: 'Sv. Fachtny z Rosscarbery, biskupa',
+    faustina_kowalska_virgin: 'Sv. Faustiny Kowalské, panny',
+    philip_of_jesus_de_las_casas_martyr: 'Sv. Filipa od Ježiša de las Casas, mučedníka',
+    ferdinand_iii_of_castile: 'Sv. Ferdinanda III. Kastilského',
+    fergal_of_salzburg_bishop: 'Sv. Virgila ze Salzburgu, biskupa a misionáře',  // TODO: `ze Salzburgu` or `solnohradského`? (src: http://librinostri.catholica.cz/download/VondruskaSvati2-OCR.pdf#page=217 … it might be an archaism though)
+    fiacre_of_breuil_monk: 'Sv. Fiakra, mnicha',  // Note: Nominative should be `Fiakr` in Czech; src: http://catholica.cz/?id=3984
+    fidelis_of_sigmaringen_priest: 'Sv. Fidela ze Sigmaringy, kněze a mučedníka',
+    finbarr_of_cork_bishop: 'Sv. Finbara, biskupa',  // Note: Nominative should be `Finbar` in Czech; src: http://catholica.cz/?id=4652
+    finnian_of_clonard_bishop: 'Sv. Finiana z Clonardu, biskupa',
+    fintan_of_clonenagh_abbot: 'Sv. Fintana, opata',
+    flannan_of_killaloe_bishop: 'Sv. Flannana, biskupa',
+    florian_of_lorch_and_companions_martyrs: 'Sv. Floriána a druhů, mučedníků',
+    florian_of_lorch_martyr: 'Sv. Floriána, mučedníka',
+    frances_of_rome_religious: 'Sv. Františky Římské, řeholnice',
+    frances_xavier_cabrini_virgin: 'Sv. Františky Xaviery Cabriniové, panny',  // TODO: `Cabriniové` or `Cabrini`? `Xaviera` or `Xavier`?
+    francis_borgia_priest: 'Sv. Františka Borgii, Priest',
```

Comment:
I’ve never heard `Borgiáš` word. :smiley: I somewhat don’t like it (because I don’t like localising/changing personal names and surnames in general), but I think you know better what is more often used or sounds better in Czech, therefore I accept your suggestion.